server/install.go

```diff
@@ -409,6 +409,7 @@ func (ip *InstallationProcess) Execute() (string, error) {
 		Labels: map[string]string{
 			"Service":       "Pterodactyl",
 			"ContainerType": "server_installer",
+			"ServerUUID":    ip.Server.ID(),
```

Comment:
Should we use something like `PdtlId` just to make it much more clear with the associated service labeling? `ServerUUID` is probably a little vague, all things considered.

I'd be open to also adding `PdtlContainerTable` as a label and deprecating `ContainerType`. Could also do `Pterodactyl` as the prefix, but that might get long and unweildly. End goal just being that these labels are tightly coupled to Pterodactyl itself and obvious at a glance without digging through this code to find what we set.